Auth0/CredentialsManager.swift

```diff
@@ -80,4 +82,11 @@ public struct CredentialsManager {
             }
         }
     }
+
+    public func credentialsOnTouch(withTitle title: String, scope: String? = nil, callback: @escaping (CredentialsManagerError?, Credentials?) -> Void) {
```

Comment:
Originally I was thinking of a `useTouch` flag in `CredentialsManager`, will do something similar.